packages/store/ts/config/v2/schema.ts

```diff
@@ -16,8 +17,27 @@ export type resolveSchema<schema extends SchemaInput<scope>, scope extends AbiTy
 
 export function resolveSchema<schema extends SchemaInput<scope>, scope extends AbiTypeScope = AbiTypeScope>(
   schema: schema,
-  scope?: scope,
+  scope: scope = AbiTypeScope as scope,
 ): resolveSchema<schema, scope> {
-  // TODO: runtime implementation
-  return {} as never;
+  return Object.fromEntries(
+    Object.entries(schema).map(([key, internalType]) => {
```

Comment:
just remembered we have a `mapObject` helper if that could clean this up

Reply:
Tried this but the types on `mapObject` are sadly not good enough, makes it such that we'd have to cast more. Punting on it for now, can revisit this later.